groovy/postInit/chemistry/ChemistryOverhaul.groovy

```diff
@@ -4640,4 +4641,105 @@ DISTILLERY.recipeBuilder()
 .fluidOutputs(fluid('water') * 1000)
 .duration(60)
 .EUt(30)
-.buildAndRegister()
\ No newline at end of file
+.buildAndRegister()
+
+//TOLUENE FROM BENZENE
+TBR.recipeBuilder()
+.notConsumable(ore('dustAluminiumChloride'))
+.notConsumable(circuit(1))
+.fluidInputs(fluid('benzene') * 50)
+.fluidInputs(fluid('chloromethane') * 50)
+.fluidOutputs(fluid('toluene') * 50)
+.fluidOutputs(fluid('hydrogen_chloride') * 50)
+.duration(10)
+.EUt(30)
+.buildAndRegister()
+
+//XYLENE FROM BENZENE
+TBR.recipeBuilder()
+.notConsumable(ore('dustAluminiumChloride'))
+.notConsumable(circuit(2))
+.fluidInputs(fluid('benzene') * 50)
+.fluidInputs(fluid('chloromethane') * 100)
+.fluidOutputs(fluid('toluene') * 50)
```

Comment:
should be xylene

Reply:
also, because methyl groups are ortho-para directors, you might want to add a material titled "ortho/para-xylene mixture"